src/Microsoft.ML.FastTree/GamModelParameters.cs

```diff
@@ -17,6 +17,7 @@
 using Microsoft.ML.Internal.Utilities;
 using Microsoft.ML.Model;
 using Microsoft.ML.Trainers.FastTree;
+using Microsoft.ML.Transforms;
```

Comment:
>Transforms [](start = 19, length = 10)

I'm seeing a surprising number of these introductions. I don't object, but I am fairly curious what changed that they were no longer necessary in the past but are necessary now.

Reply:
Good point! It is due to FeatureContributionCalculatingTransformer moving from Microsoft.ML.Data into Microsoft.ML.Transforms. 

Ml.Data was already there. 

---
In reply to: [260841129](https://github.com/dotnet/machinelearning/pull/2755#discussion_r260841129) [](ancestors = 260841129)